js/plugins/chroma/src/index.ts

```diff
@@ -280,7 +283,7 @@ export function chromaIndexer<EmbedderCustomOptions extends z.ZodTypeAny>(
           const embeddingDocs = doc.getEmbeddingDocuments(docEmbeddings);
           return docEmbeddings.map((docEmbedding, j) => {
             const metadata: Metadata = {
-              docMetdata: JSON.stringify(embeddingDocs[j].metadata),
+              docMetadata: JSON.stringify(embeddingDocs[j].metadata),
```

Comment:
That's a breaking change.... Maybe set both and somehow mark the other one deprecated?

Reply:
The other one never worked. So no one could have used it for anything.

Reply:
👍